src/util/settings.cpp

```diff
@@ -60,9 +60,16 @@ bool ReadSettings(const fs::path& path, std::map<std::string, SettingsValue>& va
     values.clear();
     errors.clear();
 
+    if (!fs::exists(path)) {
+        return true; // Ok for file to not exist
+    }
+
     fsbridge::ifstream file;
     file.open(path);
-    if (!file.is_open()) return true; // Ok for file not to exist.
+    if (!file.is_open()) {
+      errors.emplace_back(strprintf("Unable to read settings file %s. Are the permissions correct?", path.string()));
```

Comment:
Concept ACK. 

nit: Maybe error message can be changed to 

```diff
-Are the permissions correct?
+Please check permissions.
```

Reply:
I think this is a good call. The final error now presents like this: 
![image](https://user-images.githubusercontent.com/2775339/127741631-646e208d-ff01-407a-9f59-d84068c45214.png)